index.js

```diff
@@ -1,33 +1,47 @@
-const { extname } = require('path')
 const fs = require('fs').promises
 const { compileTemplate } = require('@vue/compiler-sfc')
 const { optimize: optimizeSvg } = require('svgo')
 
 module.exports = function svgLoader (options = {}) {
   const { svgoConfig, svgo } = options
 
+  const svgRegex = /\.svg(\?(raw|url|component))?$/
   return {
     name: 'svg-loader',
     enforce: 'pre',
-
+    resolveid (id) {
```

Comment:
Why is `resolveid` needed?

Reply:
If it cannot be resolved, return `null` so that Rollup displays an error.